src/state/apis/zapper/zapperApi.ts

```diff
@@ -89,7 +114,58 @@ export const zapperApi = createApi({
         return { data }
       },
     }),
+    getZapperNftUserTokens: build.query<V2NftUserItem[], GetZapperNftUserTokens>({
+      queryFn: async ({ accountIds }) => {
+        let data: V2NftUserItem[] = []
+
+        const userAddresses = accountIdsToEvmAddresses(accountIds)
+
+        for (const userAddress of userAddresses) {
+          // https://studio.zapper.fi/docs/apis/api-syntax#v2nftusertokens
+          /**
+           * docs about cursor are wrong lmeow
+           * check the length of returned items to see if there are more
+           */
+          const limit = 100
+          while (true) {
+            try {
+              const res = await zapperClient.getV2NftUserTokens({
+                // Encode query params with arrayFormat: 'repeat' because zapper api derpexcts it
```

Comment:
```suggestion
                // Encode query params with arrayFormat: 'repeat' because zapper api expects it
```

?

Reply:
😢 mang can't even blame the APIs being derp these days 5058aae59